MeetupPlatformApi/Program.cs

```diff
@@ -1,26 +1,29 @@
 using MeetupPlatformApi.Authentication.DependencyInjection;
 using MeetupPlatformApi.Persistence.DependencyInjection;
+using MeetupPlatformApi.Persistence.SwaggerDocConfiguration;
 using MeetupPlatformApi.Seedwork.Configuration;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.Configuration.AddJsonFiles(builder.Environment);
 
 builder.Services.AddControllers();
-builder.Services.AddSwaggerGen(options =>
-{
-    options.CustomSchemaIds(modelType => modelType.FullName);
-});
+
 builder.Services.AddAutoMapper(typeof(Program));
 builder.Services.AddDbContext();
 builder.Services.AddJwtAuthentication(builder.Configuration);
 
+builder.Services.AddSwaggerSwashbuckleConfiguration();
+
 var app = builder.Build();
 
 if (app.Environment.IsDevelopment())
 {
     app.UseSwagger();
-    app.UseSwaggerUI();
+    app.UseSwaggerUI(options =>
+    {
+        options.DefaultModelsExpandDepth(-1);
```

Comment:
Please, add a comment or extract this configuration to an extension method. Right now it's hard to tell, what exactly this line of code does.

Reply:
Fixed.